### ISSUE TITLE
Lower Validation mIoU than default on the CityScapes Validation Set

### ISSUE THREAD

**Issue by luislcc** [VainF/DeepLabV3Plus-Pytorch#142]
Hello, thanks for your very nice repository! I am trying to replicate the performance of the DeepLabV3+ with the ResNet Backbone on the CityScapes validation set however, I have not been able to do so by using the default hyperparameters of learning rate = 0.1, batch size = 16 and 30 000 training iterations. I am only achieving around 0.30 mIoU instead of the reported 0.76. By lowering the learning the learning rate to 0.0001 and the batch size to 2, I was able to achieve 0.645. From what I understood these weights that achieve high mIoU values were provided by members of the community, but would you perhaps be able to indicate what hyperparameters were used to obtain them? Does anyone have any idea how to achieve those performance values? Many thanks in advance!
